mesonbuild/modules/gnome.py

```diff
@@ -1161,11 +1164,20 @@ def generate_gir(self, state: 'ModuleState', args: T.Tuple[T.List[T.Union[Execut
         scan_cflags += list(self._get_scanner_cflags(dep_cflags))
         scan_cflags += list(self._get_scanner_cflags(self._get_external_args_for_langs(state, [lc[0] for lc in langs_compilers])))
         scan_internal_ldflags = []
-        scan_internal_ldflags += list(self._get_scanner_ldflags(internal_ldflags))
-        scan_internal_ldflags += list(self._get_scanner_ldflags(dep_internal_ldflags))
         scan_external_ldflags = []
-        scan_external_ldflags += list(self._get_scanner_ldflags(external_ldflags))
-        scan_external_ldflags += list(self._get_scanner_ldflags(dep_external_ldflags))
+        scan_env_ldflags = []
+        for cli_flags, env_flags in [
+            self._get_scanner_ldflags(internal_ldflags),
+            self._get_scanner_ldflags(dep_internal_ldflags),
+        ]:
+            scan_internal_ldflags += cli_flags
+            scan_env_ldflags = env_flags
+        for cli_flags, env_flags in [
+            self._get_scanner_ldflags(external_ldflags),
+            self._get_scanner_ldflags(dep_external_ldflags)
+        ]:
+            scan_external_ldflags += cli_flags
+            scan_env_ldflags = env_flags
```

Comment:
This unfortunately looks like it's been deformatted by the "black" code deformatter. Can you fix it?

Reply:
Sure, I'll submit the change with the CI fixes.

Reply:
I'd also probably change the `return` in _get_scanner_ldflags

Note the distinction is not necessarily fitting everything on one line, rather the line breaking algorithm should avoid placing code contained inside parentheses or list notation, as "indented by four lines". Notice in the above diff, that both the values inside `[]` and the indented body of the for block, blend in together.

So, this would be a better way to break across lines:
```python
        for cli_flags, env_flags in (self._get_scanner_ldflags(internal_ldflags),
                                     self._get_scanner_ldflags(dep_internal_ldflags)):
```

because both elements of the tuple are lined up, but both are to the right of the "for x, y in" part.

It also fits in 86 columns, as opposed to 131 columns.